recipes/libtinyiiod/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "libtinyiiod" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/analogdevicesinc/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0b9bc22b2e0090b905fa369fcc652fe3522a68b0aad5ded1d30a1464058596d2
+
+build:
+  number: 0
```

```suggestion
build:
  number: 0
  run_exports:
    - {{ pin_subpackage('libtinyiuod' max_pin='x.x.x') }}
```

Please add a run export which provides the correct version for ABI compatiblility with downstream packages.

---

Updated